appserver/packager/external/jakarta-ee11-shim/pom.xml

```diff
@@ -156,7 +164,45 @@
                                     org.glassfish.hk2.utilities.general;
                                     org.glassfish.hk2.utilities.reflection;
                                     org.glassfish.hk2.utilities.reflection.internal;
-                                    org.jvnet.hk2.component;version="3.0.99";shim=true
+                                    org.jvnet.hk2.component;version="3.0.99";shim=true,
+                                    <!-- weld-osgi-bundle -->
+                                    <!-- Weld API and SPI -->
+                                    org.jboss.weld.*.api.*;
+                                    org.jboss.weld.*.spi.*;
+                                    org.jboss.weld.bootstrap.event;
+                                    org.jboss.weld.context.*;
+                                    org.jboss.weld.events;
+                                    org.jboss.weld.inject;
+                                    org.jboss.weld.interceptor;
+                                    org.jboss.weld.proxy;
+                                    <!-- Weld Lite Extension Translator module -->
+                                    org.jboss.weld.lite.extension.*;
+                                    <!-- Weld modules -->
+                                    org.jboss.weld.module.ejb.*;
+                                    org.jboss.weld.module.jsf.*;
+                                    org.jboss.weld.module.jta.*;
+                                    org.jboss.weld.module.web.*;
+                                    <!-- Internal packages -->
+                                    org.jboss.weld;
+                                    org.jboss.weld.bean;
+                                    org.jboss.weld.bean.builtin;
+                                    org.jboss.weld.bean.proxy;
+                                    org.jboss.weld.bootstrap;
+                                    org.jboss.weld.bootstrap.api;
+                                    org.jboss.weld.config;
+                                    org.jboss.weld.contexts.*;
+                                    org.jboss.weld.event;
+                                    org.jboss.weld.exceptions;
+                                    org.jboss.weld.injection;
+                                    org.jboss.weld.interceptor.proxy;
+                                    org.jboss.weld.interceptor.util.proxy;
+                                    org.jboss.weld.logging;
+                                    org.jboss.weld.manager;
+                                    org.jboss.weld.module.*;
+                                    org.jboss.weld.security;
+                                    org.jboss.weld.util;
+                                    org.jboss.weld.xml;
+                                    org.jboss.weld.serialization.spi;version="5.0.99";shim=true
```

Comment:
I'm not convinced this works - I don't think we can have wildcards in the shim
```suggestion
                                    <!-- weld-osgi-bundle -->
                                    org.jboss.weld.bootstrap.api;
                                    org.jboss.weld.bootstrap.api.helpers;
                                    org.jboss.weld.construction.api;
                                    org.jboss.weld.context.api;
                                    org.jboss.weld.ejb.api;
                                    org.jboss.weld.manager.api;
                                    org.jboss.weld.servlet.api;
                                    org.jboss.weld.servlet.api.helpers;
                                    org.jboss.weld.bootstrap.spi;
                                    org.jboss.weld.bootstrap.spi.helpers;
                                    org.jboss.weld.configuration.spi;
                                    org.jboss.weld.configuration.spi.helpers;
                                    org.jboss.weld.ejb.spi;
                                    org.jboss.weld.ejb.spi.helpers;
                                    org.jboss.weld.injection.spi;
                                    org.jboss.weld.injection.spi.helpers;
                                    org.jboss.weld.interceptor.spi.metadata;
                                    org.jboss.weld.interceptor.spi.model;
                                    org.jboss.weld.resources.spi;
                                    org.jboss.weld.resources.spi.helpers;
                                    org.jboss.weld.security.spi;
                                    org.jboss.weld.serialization.spi;
                                    org.jboss.weld.serialization.spi.helpers;
                                    org.jboss.weld.servlet.spi;
                                    org.jboss.weld.servlet.spi.helpers;
                                    org.jboss.weld.transaction.spi;
                                    org.jboss.weld.bootstrap.event;
                                    org.jboss.weld.context;
                                    org.jboss.weld.context.activator;
                                    org.jboss.weld.context.bound;
                                    org.jboss.weld.context.ejb;
                                    org.jboss.weld.context.http;
                                    org.jboss.weld.context.unbound;
                                    org.jboss.weld.events;
                                    org.jboss.weld.inject;
                                    org.jboss.weld.interceptor;
                                    org.jboss.weld.proxy;
                                    org.jboss.weld.lite.extension.translator;
                                    org.jboss.weld.lite.extension.translator.logging;
                                    org.jboss.weld.lite.extension.translator.util;
                                    org.jboss.weld.lite.extension.translator.util.reflection;
                                    org.jboss.weld.module.ejb;
                                    org.jboss.weld.module.ejb.context;
                                    org.jboss.weld.module.ejb.context.beanstore;
                                    org.jboss.weld.module.jsf;
                                    org.jboss.weld.module.jsf.servlet;
                                    org.jboss.weld.module.jta;
                                    org.jboss.weld.module.web;
                                    org.jboss.weld.module.web.context.beanstore.http;
                                    org.jboss.weld.module.web.context.http;
                                    org.jboss.weld.module.web.el;
                                    org.jboss.weld.module.web.logging;
                                    org.jboss.weld.module.web.servlet;
                                    org.jboss.weld.module.web.util.el;
                                    org.jboss.weld.module.web.util.servlet;
                                    org.jboss.weld;
                                    org.jboss.weld.bean;
                                    org.jboss.weld.bean.builtin;
                                    org.jboss.weld.bean.proxy;
                                    org.jboss.weld.bootstrap;
                                    org.jboss.weld.config;
                                    org.jboss.weld.contexts;
                                    org.jboss.weld.contexts.activator;
                                    org.jboss.weld.contexts.bound;
                                    org.jboss.weld.contexts.cache;
                                    org.jboss.weld.contexts.conversation;
                                    org.jboss.weld.contexts.unbound;
                                    org.jboss.weld.event;
                                    org.jboss.weld.exceptions;
                                    org.jboss.weld.injection;
                                    org.jboss.weld.interceptor.proxy;
                                    org.jboss.weld.interceptor.util.proxy;
                                    org.jboss.weld.logging;
                                    org.jboss.weld.manager;
                                    org.jboss.weld.module;
                                    org.jboss.weld.security;
                                    org.jboss.weld.util;
                                    org.jboss.weld.xml;version="5.0.99";shim=true
```

Reply:
already added changes for the shim config, with this on my local quicklook is passing and still some issues from samples related to jersey patched version